scripts/deps-downloader.fsx

```diff
@@ -0,0 +1,18 @@
+let baseUrl = "https://github.com/MonoGame/MonoGame.Dependencies/raw/master/SDL"
+let pathToProjectBin = sprintf "../MarsBaseBuilder/bin/%s" fsi.CommandLineArgs.[2]
```

Comment:
Also, please don't get the configuration from the command line, and instead process both `Release` and `Debug`.